src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -57,9 +57,9 @@ contains the following details:
 | `--kubecontext` `<string>`            | kubeconfig context to use
 | `--repository` `<string>`             | The repository from where the various Submariner images will be sourced (default "quay.io/submariner")
 | `--version` `<string>`                | Image version
-| `--service-discovery`                 | Enable Multi Cluster Service Discovery (default true)
-| `--globalnet`                         | Enable support for overlapping Cluster/Service CIDRs in connecting clusters (default disabled)
```

Comment:
As per https://github.com/submariner-io/submariner-operator/pull/1228, we decided not to deprecate this flag.

Reply:
oops right ':D